examples/persistence_service/persistent_storage/c++11/hello_world_subscriber.cxx

```diff
@@ -84,7 +81,7 @@ int main(int argc, char *argv[])
 {
     int domain_id = 0;
     int sample_count = 0;  // Infinite loop
-    bool drs = false;
+    int is_persistent = 0;
```

Comment:
```suggestion
    bool is_persistent = false;
```

Reply:
I would prefer keeping it `int` since we use the function `atoi()` to parse the value.